test/appium/tests/test_profile.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+import time
+from tests.base_test_case import SingleDeviceTestCase
+from views.console_view import ConsoleView
+from tests import basic_user
+
+
+@pytest.mark.all
+class TestProfileView(SingleDeviceTestCase):
+
+    @pytest.mark.pr
+    def test_qr_code_and_its_value(self):
+        console_view = ConsoleView(self.driver)
+        console_view.create_user()
+        console_view.back_button.click()
+        profile_view = console_view.profile_button.click()
+        profile_view.share_my_contact_key_button.click()
+        key_value = profile_view.public_key_text.text
+        time.sleep(5)
+        profile_view.get_text_from_qr()
```

Comment:
please remove `profile_view.get_text_from_qr()` , you are calling it below in `key_value_from_qr = profile_view.get_text_from_qr()`